packages/stylelint-config-terra/CHANGELOG.md

```diff
@@ -0,0 +1,127 @@
+# ChangeLog
+
+## Unreleased
+
+* Breaking
+  * Drop support for node versions below version 10.
+  * Require stylelint version 13.
+  * Scope package under @cerner.
+
+* Changed
+  * Use jest for tests.
+
+## 3.7.0 - (July 28, 2020)
+
+Fixed
```

Comment:
The formatting for the rest of this change log is a little different than the start of this change log and our other change logs. `* Fixed` vs no asterisk `Fixed`.  Not a big deal but thought I'd point it out